src/components/Subnavigation/Subnavigation.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <nav>
+    <ul class="flex pb-3.5 border-b-1 border-gray-l">
```

Comment:
Looks like `main` still needs to be merged and the colors updated with the new conventions. (I can do that if you don't want to)